Description: fix warning when not really starting netradiant
 When doing netradian --help, you would see:
 GLib-CRITICAL **: 19:22:11.618: g_string_free: assertion 'string != NULL'
 failed -- twice. This fixes it.

diff --git a/radiant/preferences.h b/radiant/preferences.h
index 8c657fe7..a7739d4d 100644
--- a/radiant/preferences.h
+++ b/radiant/preferences.h
@@ -357,9 +357,13 @@ public:
 
 ui::Widget m_notebook{ui::null};
 
-virtual ~PrefsDlg(){
-	g_string_free( m_rc_path, true );
-	g_string_free( m_inipath, true );
+virtual ~PrefsDlg() {
+	if (m_rc_path) {
+		g_string_free( m_rc_path, true );
+	}
+	if (m_inipath) {
+		g_string_free( m_inipath, true );
+	}
 }
 
 /*!
